src/goose.rs

```diff
@@ -304,7 +304,9 @@ static APP_USER_AGENT: &str = concat!(env!("CARGO_PKG_NAME"), "/", env!("CARGO_P
 #[macro_export]
 macro_rules! task {
     ($task_func:ident) => {
-        GooseTask::new(move |s| std::boxed::Box::pin($task_func(s)))
+        GooseTask::new(std::sync::Arc::new(move |s| {
```

Comment:
we already `use std::sync::Arc;` in this file, so you can just `Arc::new()` for consistency

Reply:
We never want to use a short form in a macro ;) - I tried that and it did not end well ...

Reply:
Ah, that makes sense!